docs/src/main/asciidoc/stork-kubernetes.adoc

```diff
@@ -59,10 +59,10 @@ The various integrations in Quarkus extract the location of the service from tha
 
 == Bootstrapping the project
 
-Create a Quarkus project importing the quarkus-rest-client-reactive and quarkus-resteasy-reactive extensions using your favorite approach:
+Create a Quarkus project importing the quarkus-smallrye-stork, quarkus-rest-client-reactive and quarkus-resteasy-reactive extensions using your favorite approach:
```

Comment:
Stork is a transitive dependency. Why do you need to import it explicitly?

Reply:
Ah yes, I see `quarkus-stork` is a dependency of `quarkus-rest-client-reactive`.

Reply:
Well, it's surprising and it should not be. 

I don't now the size of Stork but it would be better for it to be an optional dependency to avoid having the dependencies (as there is multiple ones) included when not needed. 
I think a lot of users of the REST Client will not need it as they will rely to config/k8s/LB for service discovery.

Moreover the quickstart do include this dependency.

Reply:
I can open an issue to make it optional and close this PR (or put it in ice) untill it's done.

Reply:
it's how it's wired inside that makes it not possible to be an optional dependency. We discussed it when we integrate it, and we decide that it was ok. It's just a few classes.

Reply:
(it's not including the discovery provider or load balancer, just the SPI classes)